Source/Core/VideoCommon/VertexLoader_Position.h

```diff
@@ -10,11 +10,51 @@ enum class VertexComponentFormat;
 enum class ComponentFormat;
 enum class CoordComponentCount;
 
+using Common::EnumMap;
+
+using VertexPositionSizeTable =
+    EnumMap<EnumMap<EnumMap<u32, CoordComponentCount::XYZ>, ComponentFormat::Float>,
+            VertexComponentFormat::Index16>;
+
+constexpr VertexPositionSizeTable InitializeVertexPositionSizeTable()
```

Comment:
We're targeting C++20, so I think this can be `consteval` instead of `constexpr` to ensure that it's *only* used for initializing (same for any in other files).

Reply:
Done.